src/main/java/com/icthh/xm/ms/dashboard/config/ApplicationProperties.java

```diff
@@ -28,6 +28,15 @@
     private String kafkaSystemQueue;
     private String dbSchemaSuffix;
 
+    private boolean storeConfigurationEnabled;
+    private String specificationPathPattern;
+    private String specificationName;
+    private String tenantDashboardsFolderPathPattern;
+    private String tenantDashboardsFolderPath;
+    private String tenantDashboardPropertiesIdPathPattern;
+    private String tenantDashboardPropertiesIdName;
+    private Long idReservedQuantity;
+
```

Comment:
Suggest to make configuration object for dashboard and not just plain properties.
like this (yaml example)
```
storage:
     type: MSCONFIG // enum: [MSCONFG, RDBMS] default is RDBMS
     msconfig:
          specification:
               pattern: pppp
               name: nnnnn
          tenantDashboards:
               folderPathPattern
               folderPath
               propertiesIdPathPattern
               propertiesIdPath
```
or something similar but i think the idea is now clear

Reply:
added ApplicationProperties.Store properties class